SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -275,52 +264,8 @@ void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel,
 		return;
 	}
 
-	if (bIsSpatialNetDriverReady)
-	{
-		CleanupLevelInitializedNetworkActors(LoadedLevel);
-	}
-	else
-	{
-		CachedLevelsForNetworkIntialize.Add(LoadedLevel);
-	}
-}
-
-void USpatialGameInstance::CleanupLevelInitializedNetworkActors(ULevel* LoadedLevel)
-{
-	bIsSpatialNetDriverReady = true;
-	for (int32 ActorIndex = 0; ActorIndex < LoadedLevel->Actors.Num(); ActorIndex++)
-	{
-		AActor* Actor = LoadedLevel->Actors[ActorIndex];
-		if (Actor == nullptr)
-		{
-			continue;
-		}
-
-		if (USpatialStatics::IsSpatialOffloadingEnabled(GetWorld()))
-		{
-			if (!USpatialStatics::IsActorGroupOwnerForActor(Actor))
-			{
-				if (!Actor->bNetLoadOnNonAuthServer)
-				{
-					Actor->Destroy(true);
-				}
-				else
-				{
-					UE_LOG(LogSpatialGameInstance, Verbose, TEXT("This worker %s is not the owner of startup actor %s, exchanging Roles"), *GetPathNameSafe(Actor));
-					ENetRole Temp = Actor->Role;
-					Actor->Role = Actor->RemoteRole;
-					Actor->RemoteRole = Temp;
-				}
-			}
-		}
-		else
-		{
-			if (Actor->GetIsReplicated())
-			{
-				// Always wait for authority to be delegated down from SpatialOS, if not using offloading
-				Actor->Role = ROLE_SimulatedProxy;
-				Actor->RemoteRole = ROLE_Authority;
-			}
-		}
-	}
+    for (AActor* Actor : LoadedLevel->Actors)
```

Comment:
spacing here

Reply:
if only we had some kind of clang-format